src/connector/mssql.rs

```diff
@@ -34,6 +34,7 @@ pub(crate) struct MssqlQueryParams {
     connection_limit: Option<usize>,
     socket_timeout: Option<Duration>,
     connect_timeout: Option<Duration>,
+    transaction_isolation_level: Option<String>,
```

Comment:
Given isolation levels are enumerable, could we make this take an `Option<IsolationLevel>` instead?